### PR TITLE
Disable IISExpress tests

### PR DESCRIPTION
**Pull request by jkotalik**
Trying to unblock PR checks.

### CODE REVIEW

File: src/Servers/test/FunctionalTests/ResponseCompressionTests.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public ResponseCompressionTests(ITestOutputHelper output) : base(output)
         }
 
         public static TestMatrix NoCompressionTestVariants
-            => TestMatrix.ForServers(ServerType.IISExpress, ServerType.Kestrel, ServerType.Nginx,  ServerType.HttpSys)
+            => TestMatrix.ForServers(ServerType.Kestrel, ServerType.Nginx,  ServerType.HttpSys)
```
Comment (Tratcher) [inline src/Servers/test/FunctionalTests/ResponseCompressionTests.cs]: Only use comments so it's more obvious it needs to be reverted.
```suggestion
            => TestMatrix.ForServers(/*ServerType.IISExpress, */ ServerType.Kestrel, ServerType.Nginx,  ServerType.HttpSys)
```

**Review by dougbu — APPROVED** [review]
It sounds like the AzDO change is going to be rolled back, perhaps quicker than we can be sure this resolves the issue. Suggest we might not need this change.

File: src/Servers/test/FunctionalTests/ResponseCompressionTests.cs (hunk 1 above)
Comment (jkotalik) [reply src/Servers/test/FunctionalTests/ResponseCompressionTests.cs]: I don't think it matters. We are just going to do a straight revert on this no matter what.

**Comment by dougbu** [comment]
@jkotalik suggest closing this in favour of #9122

**Comment by jkotalik** [comment]
I'm going to let both of these run to completion.

**Comment by jkotalik** [comment]
Yeah there are a few more tests that failed too.

**Pull request closed without merging**